README.md

```diff
@@ -40,49 +46,145 @@ SpaceDocks build process follows the golang specifications: You need to clone th
 ---- sdb.go
 ```
 
-#### Glide
-We use the [Glide package manager](https://glide.sh) to manage the sdb dependencies. For installing glide, please execute the appropreate buildscript for your plattform (Linux/MacOS Users: build/install_glide.sh, Windows Users: build/install_glide.ps1). This will install glide into $GOPATH/bin, which requires a valid $GOPATH! It is also **required** to add $GOPATH/bin to your $PATH!
+### Get the dependencies
+
+#### Install Go
+
+Install Go 1.8. You can get it from https://golang.org/dl/ or from apt-get:
```

Comment:
SDB definitly works with Go 1.9, possibly also with Go 1.10. Maybe we should recommend that instead?